### ISSUE TITLE
Unable to run the code

### ISSUE THREAD

**Issue by BayesCausality** [JTT94/diffusion_schrodinger_bridge#1]
Hi!
Your work and repo are exciting! However, I am unable to reproduce your result successfully by following the command line you gave in the repo.

I would like to run the CelebA experiment. 

The first issue is that, I think the command line  `python data.py --data celeba --data_dir './data/` missing a apostrophe at the end?

Second is that... in celeba.py, the code cannot identify the path for the data and extra information (i.e list_landmarks_align_celeba.txt though they are not important here.).

After I change the relative path to absolute path, it starts to run. However, after 11 iterations, it reports another error from line 53 in cacheloader.py:
`RuntimeError: The expanded size of the tensor (15000) must match the existing size (8500) at non-singleton dimension 0.  Target sizes: [15000, 2, 3, 32, 32].  Tensor sizes: [8500, 2, 3, 32, 32]`.

Meanwhile, another critical issue is that, the final variance is always INF or NAN. Do you have any clue?

Again, I really appreciate that you can share the repo with the community, and I am looking forward for your reply!

**Comment by dorazhiyuyang** [comment]
the command line is 
`
python data.py --data celeba --data_dir '../data/'
`

**Comment by BayesCausality** [comment]
Exactly! After I fixed this command line there still exists errors w.r.t path in celebA.py, but it is not so critical problem anyway, I can fix it by simply modify the path on my own. 

The issue is mainly about the error happened after 11 iterations of training. 

Meanwhile I noticed that the author fixed another bug raised in this issue (INF and NAN of variance occur during training) in the latest command! Thank you!

**Comment by JTT94** [comment]
Thank you for raising this and apologies I did not see the initial message earlier

1) the data path issue will be corrected, this was an oversight as I did not want to put the full path with my username in case of anonymity issues 
2) the inf variance issue was due to some tests trying inverse time in the network, these should not have been committed to the master branch and have since been addressed
3)  I will look into the other error regarding size issue. I did not experience this and I suspect due to a bug when cache batch size is not a factor of the size of the dataset.
4) the celeba data loader was a bit naively copied from a previous paper, including the additional data files, I will check this and clean it. We did not notice an error when running, but it depends on downloading files from Google drive which may cause issues 
5) We will soon release a cleaner version of the code in Jax which is under development. There has been a lot of progress in terms of training techniques.

**Comment by BayesCausality** [comment]
Thank you! I am looking forward to your new version!

**Comment by JTT94** [comment]
Just to be clear the celeb A issue is the filepath and you need to provide the correct one, I will make this more obvious if it was not clear. I cannot reproduce your error once I correct this.
